### ISSUE TITLE
Listing blob containers does not work

### ISSUE THREAD

**Issue by rkaczka-st** [microsoft/AzureStorageExplorer#3751]
**Storage Explorer Version**: 1.15.1 (same on 1.16.0)
**Build Number**: 20200902.9
**Platform/OS**: Linux (can also be reproduced for windows)
**Architecture**: x64
**Regression From**:  not sure, may be caused by increased number of blob containers?

## Bug Description ##
going into `Local & attached/StorageAccounts/<account>/Blob Containers/Load more` Displays an error
```
Unable to retrieve child resources.

Details:
["RestError:Error \"Error: Unclosed root tag\nLine: 0\nColumn: 1677091\nChar: \" occurred while parsing the response body -
```
I cannot see any containers. I assume document have been truncated as some point.

## Steps to Reproduce ##
open `Local & attached/StorageAccounts/<account>/Blob Containers/Load more`

## Expected Experience ##
I should see a list of blob containers.

**Comment by MRayermannMSFT** [comment]
@rkaczka-st how many containers do you have in your emulator?

**Comment by rkaczka-st** [comment]
@MRayermannMSFT it wasn't emulator. 
I can see first 1924545 characters of the response and container names start with 'd' (I think those are being returned in alphabetical order).

**Comment by MRayermannMSFT** [comment]
Ok, let me open an issue against the SDK. I've seen this as well, but when listing blobs.

**Comment by MRayermannMSFT** [comment]
related: https://github.com/Azure/azure-sdk-for-js/issues/12672

**Comment by JasonYeMSFT** [comment]
@rkaczka-st The SDK bug has been fixed and we shipped the SDK including the fix in our 1.18.1. Are you able to list blob containers now?

**Comment by JasonYeMSFT** [comment]
Closing due to the original bug fixed and lack of response since then. If you still have any issue, please open a new issue instead.
